datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -541,6 +546,22 @@ enum SchemaFieldDataType {
     STRUCT
 }
 
+type EntityRelationship {
+    """
+    An AuditStamp corresponding to the last modification of this relationship
+    """
+    created: AuditStamp
+
```

Comment:
Do we need any relationship type or properties on this?

Reply:
Or is it implicit that the relationship type is "lineage"

Reply:
in the future, yes, it will make sense to add relationship type. We don't need it immediately, so I'd prefer leaving it blank for now and filling it in later as we have more context